packages/rollup-plugin-copy/src/copy.js

```diff
@@ -24,6 +24,12 @@ function copy({ patterns = [], rootDir = process.cwd() }) {
   const resolvedRootDir = path.resolve(rootDir);
   return {
     name: '@web/rollup-plugin-copy',
+    async buildStart() {
+      const files = await patternsToFiles(patterns, rootDir);
```

Comment:
Can we store the files in a variable and reuse them in `generateBundle`?

Reply:
Good idea, done.